packages/microcosm/src/action.js

```diff
@@ -38,13 +38,14 @@ class Action extends Emitter {
   children: Action[]
   revisions: Revision[]
   origin: Microcosm
+  connected: boolean
```

Comment:
i wonder if `connected` is the right variable name here. root of my suspicion here is that this means you can have a `connected` action node without any parent or children (which seems like what "connected" would imply). is something like "active" more appropriate?